x-pack/plugins/ml/public/alerting/config_validator.tsx

```diff
@@ -0,0 +1,157 @@
+
+/**
+ * Validated alert configuration
+ */
+export const ConfigValidator: FC<ConfigValidatorProps> = React.memo(
+  ({ jobConfigs = [], alertInterval }) => {
+    const resultBucketSpanInSeconds = useMemo(
+      () => resolveBucketSpanInSeconds(jobConfigs.map((v) => v.analysis_config.bucket_span)),
+      [jobConfigs]
+    );
+
+    const resultBucketSpanString =
+      resultBucketSpanInSeconds % 60 === 0
+        ? `${resultBucketSpanInSeconds / 60}m`
+        : `${resultBucketSpanInSeconds}s`;
+
+    if (jobConfigs.length === 0) return null;
+
+    const alertIntervalInSeconds = parseInterval(alertInterval)!.asSeconds();
+
+    const isAlertIntervalTooHigh = resultBucketSpanInSeconds < alertIntervalInSeconds;
+
+    const jobIssues = jobConfigs.reduce(
+      (acc, job) => {
+        if (job.state === JOB_STATE.FAILED) {
+          acc.failedJobsIds.push(job.job_id);
+        }
+        if ([JOB_STATE.CLOSING, JOB_STATE.CLOSED].includes(job.state)) {
+          acc.closedJobIds.push(job.job_id);
+        }
+        if ([DATAFEED_STATE.STOPPING, DATAFEED_STATE.STOPPED].includes(job.datafeed_config.state)) {
+          acc.stoppedDatafeedJobIds.push(job.job_id);
+        }
+        if (job.datafeed_config.state === DATAFEED_STATE.DELETED) {
+          acc.deletedDatafeedJobIds.push(job.job_id);
+        }
+        return acc;
+      },
+      {
+        failedJobsIds: [],
+        closedJobIds: [],
+        stoppedDatafeedJobIds: [],
+        deletedDatafeedJobIds: [],
+      } as Record<
+        'failedJobsIds' | 'closedJobIds' | 'stoppedDatafeedJobIds' | 'deletedDatafeedJobIds',
+        JobId[]
+      >
+    );
+
+    const configContainsIssues =
+      isAlertIntervalTooHigh || Object.values(jobIssues).some((v) => v.length > 0);
+
+    if (!configContainsIssues) return null;
+
+    return (
+      <>
+        <EuiSpacer size={'m'} />
+        <EuiCallOut
+          title={
+            <FormattedMessage
+              id="xpack.ml.alertConditionValidation.title"
+              defaultMessage="Alert condition contains the following issues:"
+            />
+          }
+          color="warning"
+          size={'s'}
+        >
+          <ul>
+            {isAlertIntervalTooHigh ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.alertIntervalTooHighMessage"
+                  defaultMessage="The check interval is higher than the result bucket span of {resultBucketSpan}. Be aware of a delay in receiving notifications."
+                  values={{
+                    resultBucketSpan: resultBucketSpanString,
+                  }}
+                />
+              </li>
+            ) : null}
+
+            {jobIssues.failedJobsIds.length > 0 ? (
+              <li>
+                <FormattedMessage
+                  id="xpack.ml.alertConditionValidation.failedJobsMessage"
+                  defaultMessage="{jobIds} {count, plural, one {job is} other {jobs are}} failed"
```

Comment:
Nit. The message above ends with a punctuation mark, so I'd add one to the rest of the messages, too.
```suggestion
                  defaultMessage="{jobIds} {count, plural, one {job is} other {jobs are}} failed."
```